recipes/metaphlan2/build.sh

```diff
@@ -8,4 +8,4 @@ utils/metaphlan2krona.py
 mkdir -p $PREFIX/bin
```

Comment:
`strainphlan.py`should also be added as a binary for 2.6.0 recipe.

Maybe also some other of the utils scripts?